docs/source/reference/kubernetes/kubernetes-setup.rst

```diff
@@ -466,6 +466,22 @@ SkyPilot supports either of two modes to expose ports:
 
 Refer to :ref:`Exposing Services on Kubernetes <kubernetes-ports>` for more details.
 
+.. _kubernetes-setup-fuse:
+
+Set up FUSE proxy
+^^^^^^^^^^^^^^^^^
+
+By default, SkyPilot automatically sets up a FUSE proxy to allow Pods created by SkyPilot to perform FUSE mounts/unmounts operations without root privileges. The proxy requires root privileges and ``SYS_ADMIN`` capabilities, which may require additional security audits. 
+
+In most clusters, SkyPilot handles setting up the FUSE proxy as a privileged DaemonSet, and **no manual configuration is required by the user**.
+
+However, if you are operating in a cluster with restricted permissions, you can deploy the DaemonSet externally to avoid granting SkyPilot to create privileged DaemonSets. SkyPilot will automatically discover the DaemonSet and use it as the FUSE proxy:
```

Comment:
```suggestion
However, if you are operating in a cluster with restricted permissions, you can deploy the DaemonSet externally, to avoid the need to grant SkyPilot the permission to create privileged DaemonSets. SkyPilot will automatically discover the DaemonSet and use it as the FUSE proxy:
```

Reply:
Thanks!